requirements/core.txt

```diff
@@ -17,4 +17,4 @@ tbats>=1.1.0
 tqdm>=4.60.0
 typing-extensions
 xarray>=0.17.0
-xgboost>=1.6.0
+xgboost>=2.1.4
```

Comment:
Since this lower limit is set on the xgboost dependency, all the logic around the `xgb_200_or_above` variable can be removed